src/mesh/mesh.cpp

```diff
@@ -211,8 +212,9 @@ Mesh::Mesh(ParameterInput *pin, ApplicationInput *app_in, Packages_t &packages,
 
   // SMR / AMR:
   if (adaptive) {
-    max_level_ref_ = pin->GetOrAddInteger("parthenon/mesh", "numlevel", 1,
-                                          "maximum level of refinement globally");
+    max_level_ref_ =
+        pin->GetOrAddInteger("parthenon/mesh", "numlevel", 1,
+                             "maximum level of refinement globally when AMR is on");
```

Comment:
This change might need a rebuilt of the autogenerated parameter docs as the string changed.